charts/sn-console/templates/console/streamnative-console-ingress.yaml

```diff
@@ -0,0 +1,62 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+{{- if .Values.ingress.enabled }}
+
+{{/* COMMENT */}}
+
+apiVersion: extensions/v1beta1
```

Comment:
`extensions/v1beta1` is deprecated in v1.14+, unavailable in v1.22+. Because it's new for sn-console, I think you should use the new apiVersion `networking.k8s.io/v1` directly

Reply:
Fixed